jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/internal/HttpChannelState.java

```diff
@@ -1120,12 +1120,12 @@ private boolean lockedIsWriting()
             return _writeCallback != null;
         }
 
-        private Runnable lockedFailWrite(Throwable x, boolean fatal)
+        private Runnable lockedFailWrite(Throwable x)
```

Comment:
maybe rename to `lockedFailIfWritePending`?

Reply:
I'd leave it with the current name.